### PR TITLE
Make Update notficiations strict and fix all inspections

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: apps/updatenotification/lib/UpdateChecker.php [hunk 1]
```diff
@@ -40,18 +41,18 @@ public function __construct(VersionCheck $updater) {
...
 		$data = $this->updater->check();
 		$result = [];
 
-		if(isset($data['version']) && $data['version'] !== '' && $data['version'] !== []) {
+		if (isset($data['version']) && $data['version'] !== '' && $data['version'] !== []) {
 			$result['updateAvailable'] = true;
 			$result['updateVersion'] = $data['versionstring'];
 			$result['updaterEnabled'] = $data['autoupdater'] === '1';
-			if(substr($data['web'], 0, 8) === 'https://') {
+			if (strpos($data['web'], 'https://') === 0) {
```
Comment (rullzer) [inline apps/updatenotification/lib/UpdateChecker.php]: No `\strpos` ;)?

File: apps/updatenotification/lib/UpdateChecker.php [hunk 2]
```diff
@@ -40,18 +41,18 @@ public function __construct(VersionCheck $updater) {
 	/**
 	 * @return array
 	 */
-	public function getUpdateState() {
+	public function getUpdateState(): array {
 		$data = $this->updater->check();
 		$result = [];
 
-		if(isset($data['version']) && $data['version'] !== '' && $data['version'] !== []) {
+		if (isset($data['version']) && $data['version'] !== '' && $data['version'] !== []) {
```
Comment (rullzer) [inline apps/updatenotification/lib/UpdateChecker.php]: `\isset` ?

**Review by rullzer — APPROVED** [review]
🚀

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
ref (#7392)

**Pull request merged**